services/server/server.go

```diff
@@ -452,6 +453,8 @@ func LoadPlugins(ctx context.Context, config *srvconfig.Config) ([]*plugin.Regis
 			ic.Meta.Exports["path"] = path
 			options := *bolt.DefaultOptions
 			options.Timeout = timeout.Get(boltOpenTimeout)
+			options.MmapFlags = syscall.MAP_POPULATE
```

Comment:
`syscall.MAP_POPULATE` is not defined for darwin/macOS, so you will need to populate these options in a platform-aware way.